requirements.txt

```diff
@@ -23,3 +23,4 @@ pydocstyle~=1.0.0
 cmakelint~=1.3.4
 requests_mock~=0.7.0
 vim-vint~=0.3.6
+nltk
```

Comment:
IIUC not providing a version would fetch the latest stable version. Anything wrong with that?